Mathlib/Analysis/Calculus/FormalMultilinearSeries.lean

```diff
@@ -159,6 +153,22 @@ end FormalMultilinearSeries
 
 end
 
+namespace FormalMultilinearSeries
+variable [CommRing 𝕜] [AddCommGroup E] [Module 𝕜 E] [TopologicalSpace E] [IsTopologicalAddGroup E]
```

Comment:
```suggestion
variable [Ring 𝕜] [AddCommGroup E] [Module 𝕜 E] [TopologicalSpace E] [IsTopologicalAddGroup E]
```
Sorry!  In my defense, this one was a little subtler, since there is no previous counterpart!  :smile: